libs/viewer/src/Settings.cpp

```diff
@@ -1171,6 +1175,7 @@ static std::ostream& operator<<(std::ostream& out, const ViewerOptions& in) {
         << "\"skyboxEnabled\": " << to_string(in.skyboxEnabled) << ",\n"
         << "\"backgroundColor\": " << (in.backgroundColor) << ",\n"
         << "\"cameraFocalLength\": " << (in.cameraFocalLength) << "\n"
```

Comment:
Add a trailing comma to cameraFocalLength.  I really need to make this automated.